SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -503,7 +557,7 @@ void USpatialSender::SendRPC(TSharedRef<FPendingRPCParams> Params)
 
 			if (!NetDriver->StaticComponentView->HasAuthority(EntityId, ComponentUpdate.component_id))
 			{
-				UE_LOG(LogSpatialSender, Verbose, TEXT("Trying to send MulticastRPC component update but don't have authority! Update will not be sent. Entity: %lld"), EntityId);
+				UE_LOG(LogSpatialSender, Verbose, TEXT("Trying to send RPC component update but don't have authority! Update will not be sent. Entity: %lld"), EntityId);
```

Comment:
I know this isn't your log but could you add to this log the function name?

Reply:
Added function name to log.